plugins/metadata/dc11/filter/Dc11SchemaPublicationFormatAdapter.php

```diff
@@ -179,7 +179,13 @@ public function extractMetadataFromDataObject(&$publicationFormat)
                 $dc11Description->addStatement('dc:identifier', $doi);
             }
         }
-
+        if ($context->areDoisEnabled()) {
+            $doi = $publication->getData('doiObject');
+            if ($doi)
+            {
+                $dc11Description->addStatement('dc:relation', $doi->getData('doi'));
+            }
+        }
```

Comment:
I would preserve the blank line between the public identifiers section and the identifier: others section just to make it more readable.

Reply:
Ok, thanx, I'll change that.